doc/modules/model_evaluation.rst

```diff
@@ -2046,48 +2046,49 @@ deviance error (D) estimated over :math:`n_{\text{samples}}` is defined as
   \text{D}(y, \hat{y}) = \frac{1}{n_\text{samples}}
   \sum_{i=0}^{n_\text{samples} - 1}
   \begin{cases}
-  (y_i-\hat{y}_i)^2, & \text{for }p=0\text{ (Normal)}\\
-  2(y_i \log(y/\hat{y}_i) + \hat{y}_i - y_i),  & \text{for }p=1\text{ (Poisson)}\\
-  2(\log(\hat{y}_i/y_i) + y_i/\hat{y}_i - 1),  & \text{for }p=2\text{ (Gamma)}\\
+  (y_i-\hat{y}_i)^2, & \text{for }\text{power}=0\text{ (Normal)}\\
+  2(y_i \log(y/\hat{y}_i) + \hat{y}_i - y_i),  & \text{for power}=1\text{ (Poisson)}\\
+  2(\log(\hat{y}_i/y_i) + y_i/\hat{y}_i - 1),  & \text{for power}=2\text{ (Gamma)}\\
   2\left(\frac{\max(y_i,0)^{2-p}}{(1-p)(2-p)}-
```

Comment:
Should we replace all these `p`s with `power` as well?

Reply:
> Should we replace all these ps with power as well?

Honestly that would be difficult to read, the formulas are complicated enough as it is.

![Screenshot_2019-09-09 3 3 Model evaluation quantifying the quality of predictions — scikit-learn 0 22 dev0 documentation](https://user-images.githubusercontent.com/630936/64550130-7cf88c80-d332-11e9-8369-9494929ef05d.png)

I think in formulas we should use `p`. I'll add a sentence to introduce it better.